jade/release-build.yaml

```diff
@@ -2,9 +2,9 @@
 # ROS buildfarm release-build file
 ---
 abi_incompatibility_assumed: true
-jenkins_binary_job_priority: 82
+jenkins_binary_job_priority: 122
```

Comment:
These priority values are out of valid range. And also invert the priority compared to other jobs. Please only change the priority values which need to be modified and only by the minimal difference (I think +1 be enough here).